### PR TITLE
Add timetrings with exact server timings

### LINKED ISSUE #277: Wrong timestamp shown after syncing message with server
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
After syncing the messages from server, Timestamp shown is wrong.

**Expected behavior:**
Timestamp should be exact.

**Steps to reproduce:**
Message something from chat.susi.ai after login and then login in Chromebot and check the timestamp

**System information:**
```
Ubuntu 17.10
```

**Would you like to work on the issue?**
Yes I am

### PR DESCRIPTION
**Pull request by ms10398**
Fixes #277 

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Timestamp strings are showing up correctly

#### Changes proposed in this pull request:
Timestamp strings are showing up correctly now

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -483,16 +485,17 @@ window.onload = function() {
...
         storageItems = items.message;
         restoreMessages(storageItems);
 
     }
-});
-
-
+    else
+    {
+	    $(".empty-history").remove();
```
Comment (gabru-md) [inline src/script.js]: Please check for indentation here.

File: src/script.js [hunk 2]
```diff
@@ -437,13 +434,18 @@ function syncMessagesFromServer() {
         chrome.storage.sync.remove("message");
         for (var i = 0; i < queryAnswerData.length; i++) {
             var query = queryAnswerData[i].query;
+            var queryTime = queryAnswerData[i].queryTime;
+            queryTime = new Date(Date.parse(queryAnswerData[i].queryTime));
+            queryTime = getCurrentTime(queryTime);
+            var answerTime = queryAnswerData[i].answerTime;
+           	answerTime = new Date(Date.parse(queryAnswerData[i].answerTime));
```
Comment (gabru-md) [inline src/script.js]: Indentation here  :)

**Comment by ms10398** [comment]
@zamhaq Please merge if appropriate.

**Comment by gabru-md** [comment]
Since it has 3 approvals.
I would say if @zamhaq does not respond in a day, then we can go with the merge?
ping @ms10398

**Pull request merged**